File: banda/SergioDeIscar-banda-java/SergioDeIscar-banda-java/src/main/java/org/example/models/Bajista.java
package org.example.models;

import org.example.interfaces.IBajista;

public final class Bajista extends Musician implements IBajista {
    private final Float salario = super.salario * 1.3f;
    private int numeroCuerdas;

    public Bajista(String nombre, int experiencia, int numeroCuerdas) {
        super(nombre, experiencia);
        setNumeroCuerdas(numeroCuerdas);
    }

    @Override
    public float getSalario() {
        return salario;
    }

    @Override
    public int getNumeroCuerdas() {
        return numeroCuerdas;
    }

    @Override
    public void setNumeroCuerdas(int numeroCuerdas) {
        this.numeroCuerdas = numeroCuerdas;
    }

    @Override
    public String getNombre() {
        return this.nombre;
    }
}

File: banda/SergioDeIscar-banda-kotlin/SergioDeIscar-banda-kotlin/src/main/kotlin/models/Bajista.kt
package bandamusical.models

import bandamusical.interfeces.IBajista

class Bajista(nombre:String,
              override val experiencia: Int,
              override val numCuerdas: Int): Musician(nombre), IBajista {
    override val salario: Float = super.salario * 1.3f
}